src/Provider/Vkontakte.php

```diff
@@ -104,33 +104,32 @@ public function getUserProfile()
 
         $data = new Collection($response->response[0]);
 
-        if (!$data->exists('id')) {
+        if (! $data->exists('id')) {
             throw new UnexpectedApiResponseException('Provider API returned an unexpected response.');
         }
 
-        $userProfile = new Profile();
+        $userProfile = new User\Profile;
```

Comment:
any reason to change it?

Reply:
@ApacheEx more readable, other providers use too call as `User\Profile`

Reply:
then, let's use it like in other providers.  `$userProfile = new User\Profile();`

Reply:
@ApacheEx pls stop this shit, it's not important